packages/keyring-controller/src/KeyringController.ts

```diff
@@ -1590,6 +1597,11 @@ export class KeyringController extends BaseController<
         throw new Error(KeyringControllerError.UnsafeDirectKeyringAccess);
       }
 
+      if (forceUpdate || !isEqual(oldKeyringState, await keyring.serialize())) {
```

Comment:
nit: As the `.serialize()` returned value will always be of type `Json`, we could do without Lodash, by matching the stringified versions - probably less overhead

Reply:
Yes that's a good question, IDK if strinfying + comparing is always faster or not? 🤔

Would probably depends on the data, but from some benchmarks I saw, `lodash` seemed to be faster!

Reply:
I think we should keep using `lodash` for now.